### PR TITLE
Chore - hbng api merge cleanup

### PR DESCRIPTION
**Pull request by clickthisnick**
*Why:*
 - Clean up some things after hbng and api was merged together

*TODO:*
  - I'm not sure how to push to the pip repository so I didn't do that

### CODE REVIEW

File: xudo/commands/build.py [hunk 1]
```diff
@@ -21,27 +19,27 @@ def run(self):
 
         if self.options["-c"]:
             os.chdir(javaPath)
-            os.system("docker rm api_changelog_1")
```
Comment (clickthisnick) [inline xudo/commands/build.py:24]: the `api_` part just comes from whatever directory the docker-compose file is in.

So now its `hbng_`

File: xudo/commands/build.py [hunk 2]
```diff
@@ -1,8 +1,6 @@
 """The logs command."""
 
 
-from json import dumps
```
Comment (clickthisnick) [inline xudo/commands/build.py:4]: I didn't see dumps being used, LMK if its needed for stdout and I can add it back in

File: xudo/commands/prune.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+from .base import Base
+
+import os
+
+
+class Prune(Base):
+    """Prompts the user to prune their docker environment"""
+
+    def run(self):
+        os.system("docker system prune")
```
Comment (clickthisnick) [inline xudo/commands/prune.py:13]: This is a nice cmd if you want to go one step further beyond just cleaning up dangling images

File: xudo/commands/logs.py [hunk 4]
```diff
@@ -13,10 +11,8 @@ class Logs(Base):
...
         if self.options["api"]:
-            os.system("docker logs api_api_1 -f")
+            os.system("docker logs hbng_api_1 -f")
         elif self.options["changelog"]:
-            os.system("docker logs api_changelog_1 -f")
+            os.system("docker logs hbng_changelog_1 -f")
         elif self.options["migrations"]:
-            os.system("docker logs api_migrations_1 -f")
-        elif self.options["rcash"]:
-            os.system("docker logs api_rcash_1 -f")
```
Comment (clickthisnick) [inline xudo/commands/logs.py:22]: rcash no longer exists

File: README.rst [hunk 5]
```diff
@@ -6,9 +6,9 @@ xudo-cli
 Installation
 ------------
 
-``pip install xudo``
+``pip install xudo`` or ``python3 setup.py install`` to install with local files
```
Comment (Ostralyan) [inline README.rst:9]: Oh yeah good call!!

File: xudo/commands/build.py (hunk 1 above)
Comment (Ostralyan) [reply xudo/commands/build.py:24]: oh didn't know this thanks.

**Review by Ostralyan — APPROVED** [review]

**Pull request merged**